src/protocols/MeterFile.cpp

```diff
@@ -79,7 +84,7 @@ MeterFile::MeterFile(std::list<Option> options)
 
 					case '%':
 						scanf_format[j++] = '%'; // add double %% to escape a conversion identifier
-                        // fallthrough
+						// nobreak;
```

Comment:
Indent?

Reply:
You‘ve reverted a gcc 7.1 fix by mbehr here- please undo this change as it breaks gcc.